internal/store/elasticsearch/discovery_search_repository.go

```diff
@@ -277,3 +293,128 @@ func (repo *DiscoveryRepository) toSuggestions(response searchResponse) (results
 
 	return
 }
+
+func (repo *DiscoveryRepository) Group(ctx context.Context, cfg asset.GroupConfig) ([]asset.GroupResult, error) {
```

Comment:
Can you add some logs at debug level in this method.

Reply:
done